docs/blog/2018-06-15-why-we-built-the-site-showcase/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Why We Created the Site Showcase
+date: 2018-06-15
+author: Shannon Soper
+tags: [""]
+---
+ 
+We recently finished the first phase of the [Site Showcase](https://next.gatsbyjs.org/showcase/), which is a searchable, filterable library of sites built with Gatsby!
+
+![Featured Sites](featured-sites.png)
```

Comment:
scroll this to the right so that it shows different sites in each screenshot, similar to 

https://twitter.com/gatsbyjs/status/1006917990309416961